spec.md

```diff
@@ -54,7 +58,7 @@ Several terms are used frequently in this document and warrant basic definitions
 - **Pull**: the act of downloading Blobs and Manifests from a Registry
 - **Blob**: the binary form of content that is stored by a Registry, addressable by a Digest
 - **Manifest**: a JSON document which defines an Artifact. Manifests are defined under the OCI Image Spec <sup>[apdx-2](#appendix)</sup>
-- **Config**: a section in the Manifest (and associated Blob) which contains Artifact metadata
+- **Config**: a blob referenced in the Manifest (and associated Blob) which contains Artifact metadata and describes the Manifest
```

Comment:
The `(and associated Blob)` here no longer makes sense.

> and describes the Manifest

Does it?

Reply:
I adjusted the language here, thanks for the suggestion.

Reply:
Do you have an update to push with the updated language?

Reply:
oops 🥵